algoliasearch/type_query.go

```diff
@@ -0,0 +1,60 @@
+package algoliasearch
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type MultipleQueriesRes struct {
+	Results []MultipleQueryRes `json:"results"`
```

Comment:
We can discuss about that but on my side, I've chosen to directly return the slice instead of this `*Res` type. Because in the end, what the user really wants are the underlying results.